pkg/operator/starter.go

```diff
@@ -442,7 +442,6 @@ var RevisionConfigMaps = []revision.RevisionResource{
 var RevisionSecrets = []revision.RevisionResource{
 	// these need to removed, but if we remove them now, the cluster will die because we don't reload them yet
 	{Name: "etcd-client"},
```

Comment:
As per the slack thread from last thursday, further investigation is required before etcd client certs can be considered safe to move to the unrevisioned path. It's not clear that long-running connections (e.g. watches) to etcd would respond in a timely manner to changes in key material.

Also, etcd client cert rotation would occur far less frequently since their duration is currently 10 years, so the potential for unwanted disruption caused by rotation is far less than for kube client certs.